### DISCUSSION TITLE
Is there a way to download an older version of Yellow and disable updates?

### DISCUSSION THREAD

**Discussion by glow12121** [datenstrom/community#767] [Ask a question] [answered]
I am asking because a recent update is only partially compatible with my older CSS theme (some things are the same, others are not) and layout (footer, for some reason, has a different style from the rest of the page).

**Comment by schulle4u** [comment, marked as answer]
Since Yellow uses a rolling release system we don't provide release archives anymore. 

Generally Yellow doesn't enforce you to install updates. If you are happy with your current version, you can stay there as long as your webhost's php environment is compatible. However sooner or later this will cause other problems. You for example won't be able to install new features or apply possible security updates to existing ones. Working together with us to solve existing problems definitely is the better way.

**Comment by glow12121** [comment]
Yes, I understand. Thank you for the answer.

I'd like to explain where I come from as a Yellow user and why I chose it, after long elaboration, over Jekyll and the like. I know basic html and css; I can delete chunks of php files I don't need - that's about all "programming" I can do. I don't run Yellow on a server but on my PC and, next, upload files via FTP.

Yellow for a long time was a make-and-forget utility for me. But after a certain update, I had to remake my style and layout as they didn't work. Then it happened again - if I am not mistaken, that time update happened automatically as I ran another command. And now this happens again when I have  to, sort of urgently, update the website. I cannot even point out elements that are wrong - the whole thing changed, more or less. That's my story : )

P. S. Also, as I mentioned in a different topic, even the settings of the previous version did not work. Can Yellow be called a rolling release if its versions are not compatible content- and settings-wise? And - I am pretty sure that my older version was not from 2020, as you put it. It should be more recent - I just don't know where to check it.

**Comment by markseuffert** [comment]
Let's solve the problem together, I'll answer you in datenstrom/community#765.
